basil/HL/sensirion_ekh4.py

```diff
@@ -86,7 +93,11 @@ def get_dew_point(self, min_val=-40, max_val=100):
             else:
                 values.append(self.cal_ret(
                     ret[2 + 2 * j] + ret[2 + 2 * j + 1]))
-        return values
+
+        if channel == 'all':
+            return values
+        elif isinstance(channel, int):
+            return values[channel]
```

Comment:
Should in return a list even if one channel?

Reply:
i don't know what's best here?

Reply:
Would return number or list. IF you return always list one cannot use return value directly and this feature would be useless, to my mind.

Reply:
Fine with me anyway if usefull.